features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Description
+
+This document describes how to update mbed OS lwip stack. Mbed OS lwip stack is a copy of
```

Comment:
- Please change "mbed OS lwip stack" to "the mbed OS LWIP stack".
- Please change the second sentence to "The mbed OS LWIP stack is a copy of the LWIP master repository."